languages/nim/exercises/concept/basics/.meta/design.md

```diff
@@ -0,0 +1,22 @@
+## Learning objectives
+
+- Know about `proc` definition
+- Know `const` definition
+- Know about type inference
+- Know about `proc` parameters
+- Know the different way of returning values from a `proc` (implicit, `return`, `result`)
+- Know how to export identifiers
+- Know how to print values for debugging (`echo`, `debugEcho`)
+
+## Out of scope
+
+- `func` (I think we'll do another exercise on this so don't what to put here)
```

Comment:
What is the difference between "Know about `proc` definition" and a `func`?

> I think we'll do another exercise on this so don't what to put here

What do you plan on doing in that other exercise? You'll probably be doing some advanced function concepts, right? If so, you can list those advanced concepts here.

Reply:
I have no plans.
`func` is closer to the mathematical definition, ie. can't have side effects like modifying global variables, writing to files etc.